native/opencv-wrapper.cc

```diff
@@ -574,4 +591,47 @@ RotatedRect cv_camshift(CvMatrix* c_bp_image, Rect crect,
     return c_rr;
 }
 
+// =============================================================================
+//   MSER
+// =============================================================================
+
+CMSER* cv_mser_new(int delta,
+                   int min_area,
+                   int max_area,
+                   double max_variation,
+                   double min_diversity,
+                   int max_evolution,
+                   double area_threshold,
+                   double min_margin,
+                   int edge_blur_size) {
+    cv::Ptr<cv::MSER> result = cv::MSER::create(delta,
+                                                 min_area,
+                                                 max_area,
+                                                 max_variation,
+                                                 min_diversity,
+                                                 max_evolution,
+                                                 area_threshold,
+                                                 min_margin,
+                                                 edge_blur_size);
+    return reinterpret_cast<CMSER*>(new cv::Ptr<cv::MSER>(result));
+}
+
+void cv_mser_drop(CMSER* cmser) {
+    cv::Ptr<cv::MSER>* mser = reinterpret_cast<cv::Ptr<cv::MSER>*>(cmser);
+    delete mser;
+    mser = nullptr;
+}
+
+void cv_mser_detectRegions(CMSER* cmser, CvMatrix* image, VecPoints* msers, VecRect* bboxes) {
```

Comment:
s/cv_mser_detectRegions/cv_mser_detect_regions?

More consistent with the rest of the naming conventions.

Reply:
I think we must follow original naming convention which is `detectRegions`. Otherwise it may be hard to find a corresponding method if you know original one.

Reply:
Ok, I don't have a strong opinion here.

Personally, I think the conversion from `OriginalMethodName` to `original_method_name` is not a hard one. It's just my preference. You don't have to change.